.github/workflows/nightly.yml

```diff
@@ -14,10 +14,11 @@ jobs:
       max-parallel: 10
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04, macos-14, windows-2019]
+        # os: [ubuntu-22.04, macos-14, windows-2019]
+        os: [ubuntu-22.04]
         python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
-        torch-version: [1.13.0, 2.0.0, 2.1.0, 2.2.0, 2.3.0, 2.4.0, 2.5.0, 2.6.0]
-        cuda-version: ['cpu', 'cu117', 'cu118', 'cu121', 'cu124', 'cu126']
+        torch-version: [1.13.0, 2.0.0, 2.1.0, 2.2.0, 2.3.0, 2.4.0, 2.5.0, 2.6.0, 2.7.0]
+        cuda-version: ['cpu', 'cu117', 'cu118', 'cu121', 'cu124', 'cu126', 'cu128']
```

Comment:
TODO: Trim some configurations to avoid:

```
.github/workflows/nightly.yml (Line: 14, Col: 7): Strategy for job 'wheel' produced 331 configurations which exceeds the maximum of 256 configurations
```
https://github.com/pyg-team/pyg-lib/actions/runs/14872140386

Reply:
Temporily disabling a few PyTorch versions that are not used as often according to https://pepy.tech/projects/torch:

<img width="300" alt="image" src="https://github.com/user-attachments/assets/7c2cc01a-0cd3-437b-aada-a8da22733fbf" />

<img width="300" alt="image" src="https://github.com/user-attachments/assets/65b7b8d5-aaf7-47f9-bdd3-e175e72afcda" />